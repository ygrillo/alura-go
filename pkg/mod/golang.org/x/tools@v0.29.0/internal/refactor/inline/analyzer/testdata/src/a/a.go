package a

func f() {
	One() // want `inline call of a.One`

	new(T).Two() // want `inline call of \(a.T\).Two`
}

type T struct{}

//go:fix inline
func One() int { return one } // want One:`goFixInline a.One`

const one = 1

//go:fix inline
func (T) Two() int { return 2 } // want Two:`goFixInline \(a.T\).Two`
