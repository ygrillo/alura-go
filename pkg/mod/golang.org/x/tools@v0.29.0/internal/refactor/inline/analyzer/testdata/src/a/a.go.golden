package a

func f() {
	_ = one // want `inline call of a.One`

	_ = 2 // want `inline call of \(a.T\).Two`
}

type T struct{}

//go:fix inline
func One() int { return one } // want One:`goFixInline a.One`

const one = 1

//go:fix inline
func (T) Two() int { return 2 } // want Two:`goFixInline \(a.T\).Two`
